src/Workspaces/Core/Portable/Remote/RemoteArguments.cs

```diff
@@ -114,7 +114,7 @@ public static SerializableSymbolAndProjectId Create(ISymbol symbol, Project proj
                 try
                 {
                     throw new InvalidOperationException(
-                        $"We should always be able to resolve a symbol back on the host side:\r\n{SymbolKeyData}\r\n{failureReason}");
+                        $"We should always be able to resolve a symbol back on the host side:\r\n{project.Name}\r\n{SymbolKeyData}\r\n{failureReason}");
```

Comment:
Do we need to worry about this from a PII standpoint? Does the exception message get sent or logged anywhere?

Reply:
this is already causing a watson crash dump.  so we're already collecting all that information.  i would presume normal watson pii takes effect here as this is all going through that system.

Reply:
Yeah, any full memory dump would already contain 'project', so the information is already in the heap. The only risk here would be triage dumps which would capture the exception but not the local variables, or if this gets logged into some _other_ system.